clippy_lints/src/methods/or_fun_call.rs

```diff
@@ -52,24 +52,15 @@ pub(super) fn check<'tcx>(
                 || (matches!(path, sym::new) && implements_default(arg, default_trait_id));
 
             then {
-                let mut applicability = Applicability::MachineApplicable;
-                let hint = "unwrap_or_default()";
-                let sugg_span = span;
-
-                let sugg: String = format!(
-                    "{}.{}",
-                    snippet_with_applicability(cx, self_expr.span, "..", &mut applicability),
-                    hint
-                );
-
+                let span_replace_word = method_span.with_hi(span.hi());
                 span_lint_and_sugg(
                     cx,
                     OR_FUN_CALL,
-                    sugg_span,
+                    span_replace_word,
```

Comment:
The `span_replace_word` name looks like that it's not `span` but something about word, so it would be better to change its name to `replace_word_span` or just put it as it is like below.
```suggestion
                    method_span.with_hi(span.hi()),
```

Reply:
done